File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;

    // Get user input for Pyramid Height
    do
    {
        n = get_int("Pyramid Height: ");
    }
    while (n < 1 || n > 8);

    //initial for loop
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j < n - i; j++)
        {
            printf(" ");
        }
    }
    printf("\n");
}

File: cs50/week6/sentimental-mario-more/mario.py
import cs50

while True:
    height = input("Height: ")
    if (height.isnumeric() == True and int(height) < 9 and int(height) > 0):
        for i in range(1, (int(height) + 1), 1):
            print(" " * (int(height) - i), "#" * (i), sep='', end='  ')
            print("#" * (i), sep='',)
        break




    sdjkhgiuashglsdifjangksdfahgiouefhglijandfghlkjfhggidfjhglijh